README.md

```diff
@@ -18,8 +18,14 @@ The server is written in elixir, enabling a distributed infrastructure. Where ea
 
 The server has the following dependencies:
 
+    Python3
     Elixir >= 1.0
-    OpenAI Gym <= 0.9.5
+    OpenAI Gym <= 0.9.
+
+To make Python3 as default Python:
+
+      $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+
```

Comment:
This way might be risky, because it maybe has some systemic errors, it depends on the user's environment. I would like to recommend to use `virtual environment`, e.g. `virtualenv -p python3 envname`. I think It is safer way. However, if both @zoq and you are fine, I'm good as well. :)

Reply:
Yeah virtualenv is better. I'll update

Reply:
Agreed.